site/content/en/docs/user/examples/injectfailure/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Inject Failure
+---
+
+# Inject Failure Demo
+
+{{< hint "info" >}}
+
+This is a demo that introduces how to inject fault to initContainer in an existing minikube cluster.
+
+{{< /hint >}}
+
+<img width="700px" src="init-container-error-injection.svg">
+
+{{< expand "Demo Detail Steps" >}}
+
+{{< code-sample file="init-container-error-injection.demo" language="bash" >}}
+
+{{< /expand >}}
+
+{{< expand "virtual-gpu-node.yaml" >}}
+
+{{< code-sample file="virtual-gpu-node.yaml" >}}
+
+{{< /expand >}}
+
+{{< expand "failed-pod.yaml" >}}
+
+{{< code-sample file="failed-pod.yaml" >}}
+
+{{< /expand >}}
```

Comment:
So are you saying I can reference each file this way?

If yes, do I specify the file path to any file I want to reference?

For example, on the `simulation.md` file I can add the below to reference the `node.yaml` file:
```md
{{< /expand >}}

{{< expand "demo/technical-outcomes/simulation/scenario-1/node.yaml" >}}

{{< code-sample file="demo/technical-outcomes/simulation/scenario-1/node.yaml" >}}

{{< /expand >}}
```

Reply:
Yes, this is because it is the same directory, so it can omit the path.

https://github.com/kubernetes-sigs/kwok/tree/main/site/content/en/docs/user/examples/injectfailure

Reply:
Lovely. Will this PR be merged today?

Reply:
Ah, I think you can copy this into your PR, I don't want this to clog up your work.

Reply:
Ah, I'll merge this #1145

Reply:
Sweet